packages/excavator-script/src/utils.ts

```diff
@@ -126,3 +130,43 @@ export function sendSpadingData(projectName: string, data: object) {
     `Excavator's project to spade ${projectName}`,
   );
 }
+
+function deleteSpadingKmail(sentTo: string): void {
+  // Check all Outbox pages once a day, then only the first page
+  const maxPage =
+    (sessionStorage.getItem("LastOutboxPurge") ?? "") !== todayToString()
+      ? Infinity
+      : 1;
+
+  let currentPage = 1;
+  while (currentPage <= maxPage) {
+    const buffer = visitUrl(
+      `messages.php?box=Outbox&begin=${currentPage}&per_page=10`,
+    ).toLowerCase();
+    if (!buffer.includes("toid")) break;
+    const messageIds: string[] = buffer
+      .split("td valign=top")
+      .filter((s) =>
+        s.match(
+          `<a href="showplayer.php\\?who=(\\d+)">${sentTo.toLowerCase()}</a>`,
```

Comment:
Is it worth getting the userid to fill in that `who=`?
do we need to make this search case insensitive? what happens if excavator gets capitalized differently?

Reply:
ahh I see, you `toLowerCase()`ed the entire string. neat!

Reply:
the entire buffer is lowercased